### PR TITLE
DWR-503 First draft at loading student and student_ethnicity tables

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: legacy_load/sql/V201708211503359336__initial_ddl.sql [hunk 1]
```diff
@@ -40,13 +40,29 @@
 **  dmg_eth_wht - White
 **  dmg_eth_2om - DemographicRaceTwoOrMoreRaces
 **  NOTE: Filipino does not exist in the legacy system
+** 12. blank or null administration_condition is treated as Valid
+** 13. not 't' complete is treated as 'Partial'
```
Comment (agorina) [inline legacy_load/sql/V201708211503359336__initial_ddl.sql]: Thanks @malaffoon for helping me with this.

File: legacy_load/sql/V201708211503359336__initial_ddl.sql [hunk 2]
```diff
@@ -40,13 +40,29 @@
...
+  message  VARCHAR(256)
+);
+
+CREATE TABLE IF NOT EXISTS dim_asmt_guid_to_natural_id_mapping (
+   guid varchar(255) NOT NULL,
+   subject varchar(4) NOT NULL,
+   grade varchar(2) NOT NULL,
+   natural_id varchar(250) NOT NULL,
+   name varchar(250) NOT NULL
+);
```
Comment (agorina) [inline legacy_load/sql/V201708211503359336__initial_ddl.sql]: This is to load data from ETS.

File: legacy_load/sql/warehouse_legacy_load.sql [hunk 3]
```diff
@@ -0,0 +1,150 @@
+USE legacy_load;
```
Comment (agorina) [inline legacy_load/sql/warehouse_legacy_load.sql]: I have only tested the SQL syntax. Once I have the data I will test the rest.

**Comment by agorina** [comment]
I am merging so that @chdietz could use the latest scripts.
Will address any comments if they come up.

File: legacy_load/sql/V201708211503359336__initial_ddl.sql (hunk 1 above)
Comment (malaffoon) [inline legacy_load/sql/V201708211503359336__initial_ddl.sql]: I'm still vacillating on this one: i find it hard to believe that nearly half of all tests in the legacy system are partial.

I want to look at the data. Perhaps there is a pattern. Like 2014-2015 tests all have blank/null complete. Or some other distribution that would make this more complicated.

File: legacy_load/sql/V201708211503359336__initial_ddl.sql (hunk 2 above)
Comment (malaffoon) [inline legacy_load/sql/V201708211503359336__initial_ddl.sql]: Are you going to checkin the CSV files they sent us? Or convert them into SQL INSERT statements?

Comment (agorina) [inline legacy_load/sql/V201708211503359336__initial_ddl.sql]: If you have time, it would help. Otherwise I will do it later. Just did not get to it yet.

**Pull request merged**